### DISCUSSION TITLE
why all the  # signs

### DISCUSSION THREAD

**Discussion by OAKIEDRIVER** [localsend/localsend#1094] [Discussion (English, International)]
why does local  send on my pc have all these #6  #2  #10  #14 and more  under the machine name

**Comment by Tienisto** [comment]
Those are the last number of your local IP address. If you have 4 numbers, then your device is connected to 4 networks. You can see them if you tap on the information button

**Reply by OAKIEDRIVER** [reply]
I am only connected to one network, my own

**Reply by Tienisto** [reply]
But you might have multiple network adapters like virtual networks. This might be the case if you have a virtual machine or some VPN

**Comment by OAKIEDRIVER** [comment]
Thank you. I have a load of numbers only on one network? My home WiFi 


     On Friday, 19 January 2024 at 13:42:18 GMT, Tien Do Nam ***@***.***> wrote:  
 
 


Those are the last number of your local IP address. If you have 4 numbers, then your device is connected to 4 networks. You can see them if you tap on the information button

**Comment by OAKIEDRIVER** [comment]
OK I see. Thanks for letting me know. I suggest option to hide them they look bad on the clean GUI

    On Sunday, 21 January 2024 at 14:07:07 UTC, Tien Do Nam ***@***.***> wrote:  
 
 


But you might have multiple network adapters like virtual networks. This might be the case if you have a virtual machine or some VPN

**Comment by ericxob77** [comment]
For me, in Send, I see numbers, but I also see names, like "Samsung" and "Windows."  What's the difference between numbers and names?

**Comment by etrigan63** [comment]
I have the same issue because of virtual networks installed by VMWare Workstation on my Linux box. This is preventing me from connecting to Localsend inside a VM. The VM is set up to bridge into my LAN.
